crates/pathfinder/src/rpc.rs

```diff
@@ -299,27 +299,6 @@ mod tests {
         static ref LOCALHOST: SocketAddr = SocketAddr::V4(SocketAddrV4::new(Ipv4Addr::LOCALHOST, 0));
     }
 
-    mod error {
-        lazy_static::lazy_static! {
-            pub static ref CONTRACT_NOT_FOUND: (i64, String) = (20, "Contract not found".to_owned());
-            pub static ref INVALID_SELECTOR: (i64, String) = (21, "Invalid message selector".to_owned());
-            pub static ref INVALID_CALL_DATA: (i64, String) = (22, "Invalid call data".to_owned());
-            pub static ref INVALID_KEY: (i64, String) = (23, "Invalid storage key".to_owned());
-            pub static ref INVALID_BLOCK_HASH: (i64, String) = (24, "Invalid block hash".to_owned());
-            pub static ref INVALID_TX_HASH: (i64, String) = (25, "Invalid transaction hash".to_owned());
-            pub static ref INVALID_BLOCK_NUMBER: (i64, String) = (26, "Invalid block number".to_owned());
-            pub static ref INVALID_TX_INDEX: (i64, String) = (27, "Invalid transaction index in a block".to_owned());
-        }
-    }
-
-    fn get_err(json_str: &str) -> (i64, String) {
-        let v: serde_json::Value = serde_json::from_str(json_str).unwrap();
-        (
-            v["error"]["code"].as_i64().unwrap(),
-            v["error"]["message"].as_str().unwrap().to_owned(),
-        )
-    }
-
```

Comment:
The important thing to highlight here is that the copied constants will be lost, to be maintained at one place (ErrorCode), we'll lose a bit of tests making sure responses are the same between versions, but also gain importance to the `ErrorCode::as_str`.